api-test/src/main/resources/pms/DeactivateFtm/DeactivateFtmNegativeScenarios.yml

```diff
@@ -0,0 +1,411 @@
+DeactivateFtmNegativeScenarios:
+   Pms_DeactivateFtm_without_Authentication_Neg:
+      endPoint: /v1/partnermanager/ftpchipdetail/deactivate-ftm
+      uniqueIdentifier: TC_PMS_DeactivateFtmNegativeScenarios_01
+      description: Deactivate FTM without Partner authentication 
```

Comment:
Add the expectation in description like 
"Deactivate FTM without Partner authentication and expecting it to fail"

Reply:
Updated the description